source/Cosmos.System2/Global.cs

```diff
@@ -61,10 +61,7 @@ public static void Init(TextScreenBase textScreen, bool initScrollWheel = true,
             // We must init Console before calling Inits.
             // This is part of the "minimal" boot to allow output.
             Debugger.Send("Creating the global console...");
-            if (textScreen != null)
```

Comment:
the reason for this was so the dev couls pass their own backend for the console (it has been "broken" for some time but)